### DISCUSSION TITLE
facing problem in loading app builder. PLZ help (Budibase cloud)

### DISCUSSION THREAD

**Discussion by osanaji** [Budibase/budibase#4284] [Help] [answered]
Is anyone facing problems using app builder from the middle east? each time i opened the design view I found this error msg (App preview failed to load An unknown error occurred). even published apps not working with error msg (There was an error loading your app The Budibase client library could not be loaded. Try republishing your app.)

Note: I created a new account and created an app from existing thems and face the same problem.

please help.

**Comment by shogunpurple** [comment, marked as answer]
Hi @osanaji - please continue discussion under https://github.com/Budibase/budibase/discussions/4283. Going to close this one as a duplicate - thanks!

**Reply by osanaji** [reply]
thank you.
